### ISSUE TITLE
Rule 5.1 needs a *good* example

### ISSUE THREAD

**Issue by ches** [alexandru/scala-best-practices#38]
[5.1. SHOULD evolve the state of actors only in response to messages received from the outside][1]

This rule shows an example of the anti-pattern but gives insufficient guidance on how to do better. This is exacerbated by the back-pressure rule, 5.4, which does precisely the anti-pattern in its code sample, albeit with an admonishing comment.

[1]: https://github.com/alexandru/scala-best-practices/blob/master/sections/5-actors.md#51-should-evolve-the-state-of-actors-only-in-response-to-messages-received-from-the-outside

**Comment by colincgn** [comment]
I would agree on a better example, I am not sure on the alternative either if this anti-pattern is an actual example from the Akka docs?

http://doc.akka.io/docs/akka/current/scala/howto.html#Scheduling_Periodic_Messages
